project/AppDependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt.*
 
 object AppDependencies {
 
-  private val bootstrapVersion = "9.18.0"
+  private val bootstrapVersion = "9.17.0"
```

Comment:
Version 9.17.0 is broken. Some tests will fail. Hence we need to use the next version 9.18.0